### DISCUSSION TITLE
Establish Active-Active replication between two AWS S3 buckets

### DISCUSSION THREAD

**Discussion by gabriellisboa** [minio/minio#17706] [Q&A] [answered]
Hello!

I must establish an active-active replication between two AWS S3 buckets in different regions, including file metadata. Does MinIO provide this feature?

_One of these buckets is located in China, so the native AWS S3 Cross-region replication is impossible._

I found MinIO docs related to Active-Active Bucket Replication between multiple MinIO deployments but none regarding two AWS S3 buckets.

Is this possible?

Thank you

**Comment by harshavardhana** [comment, marked as answer]
Active Active is with MinIO to MinIO
